contact/forms.py

```diff
@@ -204,9 +204,7 @@ def full_name(self):
 
 class BaseShortForm(forms.Form):
     comment = fields.CharField(
-        label=(
-            'If something is wrong, please give as much details as you can'
-        ),
+        label='Please give as much details as you can',
```

Comment:
it should either be "give as much detail" or "give as many details". if it's just a typo use the first one but check with someone from content as well

Reply:
true. funny thing is content asked us to say "details" :laughing: 
will raise with them